source/application/BioXAS/BioXASMainAppController.cpp

```diff
@@ -116,8 +120,25 @@ void BioXASMainAppController::onScalerConnected()
 	CLSSIS3820Scaler *scaler = BioXASMainBeamline::bioXAS()->scaler();
 
     if (scaler && scaler->isConnected() && !scalerView_) {
-	scalerView_ = new CLSSIS3820ScalerView(scaler, false);
-	mw_->addPane(scalerView_, "Detectors", "Scaler", ":/system-search.png", true);
+
+	    scalerView_ = new CLSSIS3820ScalerView(scaler, true);
```

Comment:
I'm noticing a general trend where we have widgets and then we box them together with squeeze's and add a top frame.  Maybe this could be a nice handy convenience method (something like ```createMainWindowPane(const QString &topFrameName, const QString &topFrameIcon, QWidget *widget);```?  Thoughts @davidChevrier?

@dleacock would you be interested in this task?